specification/loadtestservice/LoadTestService.Management/customOperation.tsp

```diff
@@ -0,0 +1,178 @@
+import "./models.tsp";
+import "@azure-tools/typespec-providerhub";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.ResourceManager;
+using OpenAPI;
+using Autorest;
+using Azure.ResourceManager.Foundations;
+
+namespace Microsoft.LoadTestService;
+
+#suppress "@azure-tools/typespec-azure-core/no-operation-id" "OperationId is already published"
+@get
+@doc("Lists the endpoints that agents may call as part of load testing.")
+@operationId("LoadTests_ListOutboundNetworkDependenciesEndpoints")
+@extension(
```

Comment:
Did you add this simply to be compatible with the swagger? I dont believe we need these extensions specified with TypeSpec. @markcowl is there a reason ARM specs would need to specify the openapi extensions?